specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-02-01/managedClusters.json

```diff
@@ -3687,6 +3687,11 @@
           "title": "The version of Kubernetes the Managed Cluster is running.",
           "description": "When you upgrade a supported AKS cluster, Kubernetes minor versions cannot be skipped. All upgrades must be performed sequentially by major version number. For example, upgrades between 1.14.x -> 1.15.x or 1.15.x -> 1.16.x are allowed, however 1.14.x -> 1.16.x is not allowed. See [upgrading an AKS cluster](https://docs.microsoft.com/azure/aks/upgrade-cluster) for more details."
         },
+        "currentKubernetesVersion": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The version of Kubernetes the Managed Cluster is running. If a specific Kubernetes version is specified in kubernetesVersion, currentKubernetesVersion is the same as kubernetesVersion. For example, kubernetesVersion: 1.20.13, currentKubernetesVersion: 1.20.13. If alias minor version is used in kubernetsVersion, then currentKubernetesVersion will be different. For example, kubernetesVersion: 1.20, currentKubernetesVersion: 1.20.13."
```

Comment:
```suggestion
          "description": "The version of Kubernetes the Managed Cluster is running. If a specific Kubernetes version is specified in kubernetesVersion, currentKubernetesVersion is the same as kubernetesVersion. For example, kubernetesVersion: 1.20.13, currentKubernetesVersion: 1.20.13. If kubernetesVersion does not contain a minor version, then currentKubernetesVersion will show which minor version was picked by the service. For example, kubernetesVersion: 1.20, currentKubernetesVersion: 1.20.13."
```

Reply:
I think the above is possibly more clear than "alias minor version" which I am not sure is language that customers have heard before (even though that's what we called the feature).